bin/nrnivmodl.in

```diff
@@ -50,24 +50,40 @@ if command -v xcrun; then
 fi
 
 LinkCoreNEURON=false
-if [ "$1" = "-coreneuron" ] ; then
-      LinkCoreNEURON=true
-      shift
-fi
-
 UserINCFLAGS=""
-if [ "$1" = "-incflags" ] ; then
-      UserINCFLAGS="$2"
-      shift
-      shift
-fi
-
 UserLDFLAGS=""
-if [ "$1" = "-loadflags" ] ; then
-       UserLDFLAGS="$2"
-       shift
-       shift
-fi
+UserCOREFLAGS=""
+
+# - options come first but can be in any order.
+while test -n "$1" ; do
+    opt="$1"
+    case "$opt" in
+    -coreneuron)
+        # also run nrnivmodl-core
+        LinkCoreNEURON=true
+        shift;;
+    -incflags)
+        # extra include flags and paths (NEURON only)
+        UserINCFLAGS="$2"
+        shift
+        shift;;
+    -loadflags)
+        # extra link flags, paths, libraries (NEURON only)
+        UserLDFLAGS="$2"
+        shift
+        shift;;
+    -l)
+        # extra lin flags, paths. libraries (CoreNEURON)
+        UserCOREFLAGS="$UserCOREFLAGS -l $2"
+        shift
+        shift;;
```

Comment:
@nrnhines : I am wondering why we need new CLI flag here. Why can't we just pass `-loadflags` to coreneuron? 🤔 
Even neuron will have some extra include/link flags, its perfectly fine to use those for coreneuron as well?

Reply:
I did change in https://github.com/neuronsimulator/tqperf/pull/8 and ea268d953822f7685b4e5e4d36592892354ba7ad.